lib/worldwide/region.rb

```diff
@@ -197,9 +198,12 @@ class Region
     # If true, then the province is optional for addresses in this region.
     attr_accessor :province_optional
 
-    # A hash of additional address fields and the rules for concatening them into the standard fields
+    # An array of additional address fields that are required for this region
```

Comment:
Not all fields are required, at least in the sense that we employ. How about _address fields that are defined_... ?